include/swift/Frontend/ParseableInterfaceSupport.h

```diff
@@ -27,6 +27,10 @@ class ModuleDecl;
 
 /// Options for controlling the generation of the .swiftinterface output.
 struct ParseableInterfaceOptions {
+  /// Should we prefer printing TypeReprs when writing out types in a module
+  /// interface, or should we fully-qualify them?
+  bool PreserveTypesAsWrittenInModuleInterface = false;
```

Comment:
I misread this as "types [as written in the module interface]" instead of "types [as written] [in the module interface]". Since it's already in ParseableInterfaceOptions, I think you can just drop the last part.